backend/app/data_processing/color.py

```diff
@@ -0,0 +1,49 @@
+import numpy as np
+
+from app.synthesis import synthesizers as synths
+
+
+def generate_samples(colors):
+    '''
+    Converts a list of colors into a list of tones.
+
+    :colors: list of dictionaries containing the number properties h, s, v
+    :return: list of audio samples corresponding to each color
+    '''
+    samples = []
+    for hsv in colors:
+        h,s,v = hsv['h']/360, hsv['s']/100, hsv['v']/100
```

Comment:
Very cool! Much easier to understand what's going on in the frontend now.

I think we might want to using hue for the harmonic weights rather than value. S and V would be pretty easy to understand as mappings to gain and freq, and then the most complex parameter (hue) gets assigned to the thing that seems most like "color" in the sound.